archetype/src/main/resources/archetype-resources/Dockerfile

```diff
@@ -89,14 +89,14 @@
 ## Once Open Liberty has a proper release for EE 11, this should be removed.
 #if (${jakartaVersion} == '11')
 #set ($baseImage = "icr.io/appcafe/open-liberty:beta")
+#elseif (${javaVersion} == '21')
+#set ($baseImage = "icr.io/appcafe/open-liberty:full-java21-openj9-ubi-minimal")
 #elseif (${javaVersion} == '17')
-#set ($baseImage = "open-liberty:full-java17-openj9")
+#set ($baseImage = "icr.io/appcafe/open-liberty:full-java17-openj9-ubi")
 #elseif (${javaVersion} == '11')
-#set ($baseImage = "open-liberty:full-java11-openj9")
-#elseif (${javaVersion} == '8')
```

Comment:
Please put back the Java SE 8 in addition to the default catch-all case.

Reply:
Looks like for the catch all default case, we could just leave out the tag. This is just a failsafe anyway.

Reply:
I'm not sure what the best way is to handle the fallback. Options:
1. Version on master: Java 8 -> open-liberty:full-java8-openj9, fallback: open-liberty:full. But Docker tag "full" doesn't exist in icr.io. Note that both tags reference the same Docker image
2. Fail template generation on fallback. This would be my preferred option, as it would detect this problem (no image defined for Java 21). But I didn't find a way to cleanly fail in Velocity
3. Current version: Include Java 8 in fallback
4. Explicit fallback: Java 8 -> full-java8-openj9-ubi, fallback -> full-java8-openj9-ubi

Options 1., 3. and 4. result in the same Docker image being used, so it's a matter of style.

Reply:
Please leave out the tag for the fallback case. That will resolve to “latest”. As I mentioned, this is just a failsafe. Hopefully it’s a path that is never even executed. This will keep this code block consistent with the rest.